pkg/util/validation/limits_test.go

```diff
@@ -776,11 +813,16 @@ func TestExtensions(t *testing.T) {
 	})
 
 	t.Run("default limits does not interfere with tenants extensions", func(t *testing.T) {
+		// Reset the default limits at the end of the test.
+		t.Cleanup(func() {
+			SetDefaultLimitsForYAMLUnmarshalling(getDefaultLimits())
```

Comment:
_I left this comment on the individual commit too, but it didn't show on the PR_

Should we move this closed to the other `SetDefaultLimitsForYAMLUnmarshalling` so it's easier to spot what's being cleaned up. Currently they're 10 lines apart

Reply:
I think it's a good idea. Done here: https://github.com/grafana/mimir/pull/7030/commits/0077fd1afd363d2cad46c235276b0d07e04e87d0